notebooks/vertex_genai/solutions/vertex_llm_tuning.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p>Blue -&gt; BLEU (usual in upper case)</p><p>Rouge -&gt; ROUGE</p><p> </p><p>I think the BLEU and ROUGE should be elaborated a bit more. The descriptions looks almost the same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Good catch! Done!</p>